### ISSUE TITLE
[Suggestion] Allow to set collision behaviour against layers for each collider, and have an option for collisions between layers to behave like triggers.

### ISSUE THREAD

**Issue by Menotdan** [FlaxEngine/FlaxEngine#1834]
It would be nice to be able to configure collisions with layers on a per-collider basis, not just in the Physics Settings file. In unreal you have the following per-collider:
![image](https://github.com/FlaxEngine/FlaxEngine/assets/32620310/f306a07a-09be-425e-9673-6ce6f8b6a4ac)
(Overlap is basically a trigger in Flax)

And also, as shown in the screenshot, having an option for two layers to interact as triggers instead of colliders could be useful.

**Comment by RuanLucasGD** [comment]
I think new versions of Unity have this feature too

**Comment by Chikinsupu** [comment]
> I think new versions of Unity have this feature too

Not quite. They have an option to include/exclude layers per collider. 
![image](https://github.com/FlaxEngine/FlaxEngine/assets/6757167/7dfd785a-502a-479f-bee4-fd520eac9885)

which would be easier to implement in flax, since the physics system is pretty close to unity's
